### PR TITLE
fixes bug 1216807 - Get rid of funfactory

### PR DESCRIPTION
**Pull request by peterbe**
This might need some care when reviewing. I'm quite confident about the code changes, but what I'm less confident about is the django-compressor stuff. 

I was able to set
```
COMPRESS_ENABLED=False
COMPRESS_OFFLINE=False
```
and run the site fine. The `.less` files were converted to `.css` just fine. 

Also, I was able to change to 
```
COMPRESS_ENABLED=True
COMPRESS_OFFLINE=True
```
and run
```
./manage.py collectstatic --noinput && ./manage.py compress --engine=jinja2 --force
```
like I always do and then runserver and that worked fine too.

### CODE REVIEW

**Comment by peterbe** [comment]
Actually, I need to review how the wsgi file works. I see some import of `manage` there which I don't think we should need. I would love to see this resolved first though https://bugzilla.mozilla.org/show_bug.cgi?id=1217184#c1

**Comment by peterbe** [comment]
@AdrianGaudebert merge conflict fixed.

File: webapp-django/crashstats/settings/base.py [hunk 1]
```diff
@@ -543,6 +609,18 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
...
+COMPRESS_CSS_FILTERS = (
+    'compressor.filters.css_default.CssAbsoluteFilter',
+    'compressor.filters.cssmin.CSSMinFilter'
+)
+
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'compressor.finders.CompressorFinder',
+)
```
Comment (AdrianGaudebert) [inline webapp-django/crashstats/settings/base.py]: These are set twice in this file (the first occurrence being around line 100).

**Comment by AdrianGaudebert** [comment]
This is looking good. I've been browsing around with these changes and saw no problem. Hopefully stage will accept those changes gladly as well... :)

r+

**Pull request merged**